[DOC] Use consistent project names everywhere (#436)
process-analytics/bpmn-visualization-examples: README.md

You can check the [examples projects](#projects) in this repository or the [Live IDE examples](#live_ide) to know how to bootstrap `bpmn-visualization` in a project.

TypeScript's users should also read the paragraph about the [TypeScript support in the `bpmn-visualization` README](https://github.com/process-analytics/bpmn-visualization-js/#-typescript-support)
especially when using `bpmn-visualization` prior version `0.27.0`. 

## 👁️‍🗨️ Demos

- [Load and Navigation demo](demo/load-and-navigation/index.html) - shows several features of `bpmn-visualization` at the same time. The sources of the demo are available in the

>   [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.

Author: Because it the name of the repository here and not of the library 😉

>   [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.

Reviewer: IHMO, we should use the bpmn-visualization name and never use bpmn-visualization-js (or whatever the repository is named on GitHub). This introduces a new name and this is exactly what we want to avoid here. AFAIK, we want a single name when we are talking about the TS lib even if we are talking about its repository or npm package.

If you prefer, we could use "in the repository of [bpmn-visualization].."

>   [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.

Author: I'm ok for your proposition 🙂

>   [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.

Author: So, I think that putting the link on the word **repository** is more appropriate. What do you think about it?

```suggestion
  [repository](https://github.com/process-analytics/bpmn-visualization-js) of `bpmn-visualization-js` .
```

>   [bpmn-visualization-js](https://github.com/process-analytics/bpmn-visualization-js) repository.

Reviewer: Fine! 😄

- [Hacktoberfest themes](demo/hacktoberfest-custom-themes/README.md) - special Hacktoberfest diagram with Hacktoberfest colors
- [Monitoring of all process instances demo](demo/monitoring-all-process-instances/README.md) - show how to use `bpmn-visualization` to render the monitoring of all process instances for a defined process

> - [Demo for ICPM 2022](demo/icpm-2022/README.md) - show a **Process Mining** scenario (_Conformance_, _Compliance_ and _Happy path_)

Author: For me, bold text is not a warning, but just a way to highlight a part of the text to make it more readable.

I find the formula `a Process Mining scenario` too long to read.
If you don't want to put in bold, I would suggest `a scenario of Process Mining` instead.

> - [Demo for ICPM 2022](demo/icpm-2022/README.md) - show a **Process Mining** scenario (_Conformance_, _Compliance_ and _Happy path_)

Reviewer: My remark about bold is for consistency "So far, bold is used for warnings in this README file."
As this change is not related to the topic, please drop it and create a new PR if you want to debate.

> - [Demo for ICPM 2022](demo/icpm-2022/README.md) - show a **Process Mining** scenario (_Conformance_, _Compliance_ and _Happy path_)

Author: ```suggestion
- [Demo for ICPM 2022](demo/icpm-2022/README.md) - show a Process Mining scenario (_Conformance_, _Compliance_ and _Happy path_)
```

## 📃 License

> The project of `bpmn-visualization` examples is released under the [Apache 2.0](LICENSE) license. \

Author: ```suggestion
The content of the `bpmn-visualization-examples` repository is released under the [Apache 2.0](LICENSE) license. \
```
